tests/yam/yast_modules/control_center.pm

```diff
@@ -10,19 +10,28 @@
 use base 'y2_installbase';
 use strict;
 use warnings;
-use testapi qw(select_console enter_cmd wait_still_screen save_screenshot get_var);
+use y2_module_basetest;
+use testapi;
 
 sub run {
-    my $podman_cmd = "podman container runlabel run registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/yast-mgmt-ncurses-test-api";
+    my $podman_cmd = " podman container runlabel run registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/yast-mgmt-ncurses-test-api;";
     select_console('root-console');
-    enter_cmd(get_var('YUI_PARAMS') . " $podman_cmd");
+    my $podman_start = y2_module_basetest::with_yast_env_variables(get_var('YUI_PARAMS')) . $podman_cmd;
+    if (!script_run($podman_start . " echo podman-status-\$? > /dev/$serialdev", 0)) {
```

Comment:
I'm not sure use the return value of script_run to judge the podman cmd success to quit or not is correct or not.
I think normally we should run wait_script and loop wait the output of serial to get podman-status-0 then test pass.